src/sql/workbench/services/notebook/browser/models/notebookModel.ts

```diff
@@ -450,6 +469,33 @@ export class NotebookModel extends Disposable implements INotebookModel {
 		return cell;
 	}
 
+	/* Adds Paramters cell based on Notebook URI parameters */
```

Comment:
This should have the parameters too per normal js doc formatting - if you type `/**` and hit enter in the line above the function it should generate the template for you

Reply:
gotcha thanks for letting me know.